pages/common/vdir.md

```diff
@@ -0,0 +1,32 @@
+# vdir
+
+> List directory contents.
+> Drop-in replacement for `ls -l`.
+
+- List files in the current directory, one per line, with details:
+
+`vdir`
+
+- List files with size displayed using human readable units (KB, MB, GB):
```

Comment:
Perhaps this will read better?

```suggestion
- List files with sizes displayed in human readable units (KB, MB, GB):
```

Reply:
I just used the same wording [as the `ls` command](https://github.com/tldr-pages/tldr/blob/master/pages/common/ls.md), but yeah I guess this could be updated if you think is clearer. It is clear either way to me so I can go with whichever you prefer.

Reply:
I think my suggestion is a little clearer.

Reply:
Applied :+1: